src/test-run/commands/observation.js

```diff
@@ -58,3 +58,9 @@ export class DebugCommand {
     }
 }
 
+export class DisableDebugCommand {
```

Comment:
Should this command be inherited from the `CommandBase` as described in the type declaration?

Reply:
I see that we have some inconsistency between our CommandBase declaration and some other commands. We have testRun.executeCommand(command: BaseCommand) method which requires command of the BaseCommand type. At the same time we have a lot of Commands which are not inherited from CommandBase.

I think all commands should be inherited from CommandBase including DebugCommand and DisableDebugCommand